ci/jobs/analytical-dataset-generation-dev.yml

```diff
@@ -1,10 +1,19 @@
 jobs:
-  - name: analytical-dataset-generation-dev
-    plan:
-      - get: aws-analytical-dataset-generation
-        trigger: true
-      - get: emr-launcher-release
-        trigger: true
-      - .: (( inject meta.plan.terraform-bootstrap ))
-      - .: (( inject meta.plan.terraform-apply ))
-      - .: (( inject meta.plan.terraform-plan ))
+- name: analytical-dataset-generation-dev
+  plan:
+  - aggregate:
+    - get: aws-analytical-dataset-generation
+      trigger: true
+    - get: emr-launcher-release
+      trigger: true
+  - .: (( inject meta.plan.terraform-bootstrap ))
+  - .: (( inject meta.plan.create-aws-profiles ))
+    config:
+      params:
+        AWS_ACC: ((aws_account.development))
+  - .: (( inject meta.plan.terraform-apply ))
+    params:
+      TF_WORKSPACE: dev
```

Comment:
```suggestion
      TF_WORKSPACE: default
```

Reply:
Can we just remove `TF_WORKSPACE` from the dev pipeline. I don't think we set it in other dev pipelines, do we?